client/landing/jetpack-cloud/components/date-picker/style.scss

```diff
@@ -0,0 +1,10 @@
+.date-picker .button,
+.date-picker__current-display-date,
+.date-picker .date-range {
+    display: inline-block;
+    float: none;
+}
+
+.date-picker .button .gridicon.disabled {
+    fill: #dcdcde;
```

Comment:
Can this colour come from a variable? Is there one that could be used here?

Reply:
Doesn't look like Gridicon has a color prop. Will definitely address styling in a subsequent PR though.